src/test/java/com/marklogic/spark/writer/WriteRowsTest.java

```diff
@@ -34,6 +38,36 @@ public class WriteRowsTest extends AbstractIntegrationTest {
 
     private final static String EXPECTED_COLLECTION = "my-test-data";
 
+    @Test
+    void streaming() throws Exception {
+        StreamingQuery stream = newSparkSession().readStream()
+            .schema(new StructType()
+                .add("docNum", DataTypes.IntegerType)
+                .add("docName", DataTypes.StringType))
+            .option("header", true)
+            .format("csv")
+            // Spark complains if this is a single file path - ??
```

Comment:
I didn't dig into why this is - but Spark complained if I did "data.csv", but is fine with "data*.csv".

Reply:
Spark is expecting a directory here when stream is used.